Feature

jamestucker1988 opened the pull request.

jamestucker1988 commented:
the project required us to make a responsive site using modern techniques like grid, flex and float.the project stressed to make to break points as such I have them in my project.

jamestucker1988 commented:
I could not figure out my linter/stylelint error. In my local machine there is no stylelint errors. please help me.

elmejdki commented on README.md:
```diff
@@ -1 +1,46 @@
-# tnw
\ No newline at end of file
+# Project 5 - smashing_magazine_clone
+​
+This is an assigned project where we were supposed to replicate/clone smashingmagazine page as seen in the link:
+<a href="https://www.smashingmagazine.com/">click here</a>
+​
+​
```
- please your readme file professional, this is not a smashing magazine clone. fix the links also

elmejdki commented on README.md:
```diff
@@ -1 +1,46 @@
+​
+​
+
+## Preview of our webpage
```
remove the unused titles.

bolah2009 commented on .github/workflows/linters.yml:
```diff
@@ -0,0 +1,48 @@
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x
```
Can you please update your config to the recent configuration? 🙂
This should solve the issue you are having. Sorry for the inconvenience. 🙏🏾

Please check here https://github.com/microverseinc/linters-config/blob/07433021cd211c0a22c51a2cb797e86e25d2dde2/html-css/.github/workflows/linters.yml#L45


## 💡 THIS IS NOT A CRR

bolah2009 replied:
@jamestucker1988 the config file, you should change is here 👇 (**.github/workflows/linters.yml**)

https://github.com/jamestucker1988/tnw/blob/7838b7da0dc0a1fed4ba4bbd6155717068216249/.github/workflows/linters.yml#L45

```diff
-          npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x
+          npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator
```

jamestucker1988 replied:
thank u so much bolah . I greatly appreciate ur help.  otherwise, I would have been stuck there.

jamestucker1988 commented:
the project required us to make a responsive site using modern techniques like grid, flex and float.the project stressed to make to break points as such I have them in my project.

expjazz commented on README.md:
```diff
@@ -1 +1,63 @@
-# tnw
\ No newline at end of file
+# Project 5 - smashing_magazine_clone
```
The title is wrong.

jamestucker1988 commented:
the project required us to make a responsive site using modern techniques like grid, flex and float.the project stressed to make to break points as such I have them in my project.

jamestucker1988 commented:
The project required us to make a responsive site using modern techniques like grid, flex and float.the project stressed to make to break points as such I have them in my project.

**Here is screen shot of webpage (4 row of images on my desktop)**<a href="images/tnwscreenshot.png">click here</a>

krys2fa commented:
## Great job implementing the changes requested, @jamestucker1988 👍  
Your project displays good use of semantic HTML as well as good CSS best practices. 💪 

Just a few more adjustments to complete the project.
### Status: Changes Required ♻️ 

Please note that these changes are requested at a breakpoint of 768px and 1024px
- [ ] The icons in the footer have rounded borders and the links below them are all capitalized.

Original Page          |  Your Page
:-------------------------:|:-------------------------:
![Screen Shot 2020-07-19 at 7 37 58 AM](https://user-images.githubusercontent.com/18187039/87869895-105b0f00-c993-11ea-9013-df4454826846.png) |  ![Screen Shot 2020-07-19 at 7 38 15 AM](https://user-images.githubusercontent.com/18187039/87869902-17821d00-c993-11ea-9834-5a5ced3f54b3.png)


- [] Your page looks different on both breakpoints. Below are the differences at breakpoint 768px.

Original Page          |  Your Page
:-------------------------:|:-------------------------:
![Screen Shot 2020-07-19 at 7 43 49 AM](https://user-images.githubusercontent.com/18187039/87869993-b0189d00-c993-11ea-99ce-d784181ed155.png) |  ![Screen Shot 2020-07-19 at 7 44 03 AM](https://user-images.githubusercontent.com/18187039/87870005-bad33200-c993-11ea-97c5-f341eac5a153.png)


#### HINT: You can use your browser's developer tools to test the appearance of your clone across multiple screen resolutions.
You can then compare your changes with the original site.

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 
Chris Amanor (Slack)

jamestucker1988 commented:
fixed the changes mentioned after review.
1)footer icons have rounded borders.
2)the breakpoint issue has been adjusted.

jamestucker1988 commented:
### THE NEXT WEB CLONE
- Introduced the breakpoint 1024 px in the media query.
-styled background of footer icons.
- adjusted the latest-deal section.
-adjusted the latest-news section.
-adjusted nav-bar.

jamestucker1988 commented:
### THE NEXT WEB CLONE.
-In this pull request I have fixed the following enumerated in bullets.
- On the top 3 news, the TWN logo disappears on medium size:
image

-On the Latest Funding Section, the image should be inside the boxes for all sizes:
image
-The text from Latest Founding Section overlaps with the categories section when close to medium size.
-Text is not properly aligned inside the images on the Sections links.

jamestucker1988 commented:
### THE NEXT WEB CLONE.
-In this pull request, I have fixed the following enumerated in bullets
-At 756 px, the menu bar and the logo have been fixed.
-improved the header section at 756px width.
-Improved font-size of the funding section.

Spayco commented on README.md:
```diff
@@ -1 +1,56 @@
+Give a ⭐️ if you like this project!
+​
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+  ​
```
- [ ] either fill these or just delete the section

Spayco commented on index.html:
```diff
@@ -0,0 +1,926 @@
+                aria-hidden="true"
+              ></span></h3
+          ></a>
+          <div class="high-five">
+            <h5>Add info</h5>
+            <h5>|</h5>
+            <h5>Get updates</h5>
+            <h5>|</h5>
+            <h5>see More</h5>
+          </div>
+        </div>
```
- [ ] on the medium and small screens, the texts are flipped 
website:
![image](https://user-images.githubusercontent.com/11758151/90808846-8f3bc280-e318-11ea-993a-705f93bd8dea.png)
![image](https://user-images.githubusercontent.com/11758151/90808925-aaa6cd80-e318-11ea-8a69-b836cdaf34a3.png)

yours:

![image](https://user-images.githubusercontent.com/11758151/90808993-c27e5180-e318-11ea-9cfd-363b922c0f3f.png)
![image](https://user-images.githubusercontent.com/11758151/90809015-cca05000-e318-11ea-9338-4a16763d492a.png)

Spayco reviewed (changes requested):
# Changes Requested 

Hello There! 👋 

Awesome job so far!! 👍 you're few steps closer to approval

## Required Changes ♻️ 

* [ ] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)), instead of commenting before every review, simply put the description on your first post in the PR

- [ ] your page has some horizontal overflow, please fix it 
![image](https://user-images.githubusercontent.com/11758151/90808536-163c6b00-e318-11ea-945d-72be76a3b748.png)

- ** I included more requested changes below**

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

jamestucker1988 commented:
This project is a curriculum-based work where we have to clone  thenextweb.com. 
- I have used modern css techniques like grid and flex to style the page.
-appropriate use of semantics is considered.
-the project is divided into nav, main-section, footer with sub-section.

Spayco reviewed (approved):
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!

Tip: From now on please include your description in the top post, not in a separate comment


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_
